src/main/java/org/candlepin/guice/CandlepinModule.java

```diff
@@ -481,52 +471,38 @@ protected void configureModelTranslator() {
+    @Provides
+    @Singleton
+    @Named("ActivationListenerObjectMapper")
     private ObjectMapper configureActivationListenerObjectMapper() {
-        ObjectMapper mapper = new ObjectMapper();
-        AnnotationIntrospector primary = new JacksonAnnotationIntrospector();
-        AnnotationIntrospector secondary = new JaxbAnnotationIntrospector(mapper.getTypeFactory());
-        AnnotationIntrospector pair = new AnnotationIntrospectorPair(primary, secondary);
-        mapper.setAnnotationIntrospector(pair);
-
-        return mapper;
+        return ObjectMapperFactory.getActivationListenerObjectMapper();
     }
 
-    @Provides @Singleton @Named("EventFactoryObjectMapper")
+    @Provides
+    @Singleton
+    @Named("EventFactoryObjectMapper")
     private ObjectMapper configureEventFactoryObjectMapper() {
-        ObjectMapper mapper = new ObjectMapper();
-
-        SimpleFilterProvider filterProvider = new SimpleFilterProvider();
-        filterProvider.setFailOnUnknownId(false);
-
-        mapper.setFilterProvider(filterProvider);
-
-        Hibernate5Module hbm = new Hibernate5Module();
-        hbm.enable(Hibernate5Module.Feature.FORCE_LAZY_LOADING);
-        mapper.registerModule(hbm);
-        mapper.registerModule(new Jdk8Module());
-        mapper.registerModule(new JavaTimeModule());
-
-        AnnotationIntrospector primary = new JacksonAnnotationIntrospector();
-        AnnotationIntrospector secondary = new JaxbAnnotationIntrospector(mapper.getTypeFactory());
-        AnnotationIntrospector pair = new AnnotationIntrospectorPair(primary, secondary);
-        mapper.setAnnotationIntrospector(pair);
-
-        return mapper;
+        return ObjectMapperFactory.getEventFactoryObjectMapper();
     }
 
-    @Provides @Singleton @Named("X509V3ExtensionUtilObjectMapper")
+    @Provides
+    @Singleton
+    @Named("X509V3ExtensionUtilObjectMapper")
     private ObjectMapper configureX509V3ExtensionUtilObjectMapper() {
-        ObjectMapper mapper = new ObjectMapper();
-        mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
-        mapper.setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY);
-        return mapper;
+        return ObjectMapperFactory.getX509V3ExtensionUtilObjectMapper();
     }
 
-    @Provides @Singleton @Named("HypervisorUpdateJobObjectMapper")
+    @Provides
+    @Singleton
+    @Named("HypervisorUpdateJobObjectMapper")
     private ObjectMapper configureHypervisorUpdateJobObjectMapper() {
-        ObjectMapper mapper = new ObjectMapper();
-        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-        return mapper;
+        return ObjectMapperFactory.getHypervisorUpdateJobObjectMapper();
+    }
+
+    @Provides
+    @Singleton
+    @Named("SyncUtilsObjectMapper")
+    private ObjectMapper configureSyncUtilsObjectMapper() {
+        return ObjectMapperFactory.getSyncUtilsObjectMapper(this.config);
     }
```

Comment:
We probably want to bind the ObjectMapperFactory as an unnamed provider to handle the cases where we don't need any special, named configuration (i.e. RuleObjectMapper). Then we can eliminate any direct static invocation of the factory method(s).